### DISCUSSION TITLE
Is there any Figma file available for this launcher?

### DISCUSSION THREAD

**Discussion by imrtls00** [MM2-0/Kvaesitso#1123] [Q&A] [answered]
I want to work on improving the whole UX of the app, 
- slight tweaks, 
- overall consistency, 
- easy access to frequently used settings/items.

If we have a file already, please share, if not, I'll have to start from scratch :)
CC: @MM2-0

**Comment by MM2-0** [comment, marked as answer]
There isn't, I rarely create mock ups.

Feel free to make suggestions, but no promises :)

**Reply by imrtls00** [reply]
Totally understandable!
I'll work on this in my free time, 

& keep up the amazing work 🎉

**Reply by brahm0** [reply]
Absolutely fantastic! I like the simplicity and thoughts behind it. Please keep it up. 🙏🙏
